org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -463,6 +463,11 @@ public class Preferences {
 	public static final String JAVA_COMPILE_NULLANALYSIS_NONNULL = "java.compile.nullAnalysis.nonnull";
 	public static final String JAVA_COMPILE_NULLANALYSIS_NULLABLE = "java.compile.nullAnalysis.nullable";
 
+	/**
+	 * Preference key for list of cleanups to run on save
+	 */
+	public static final String JAVA_CLEANUPS_ENABLED = "java.cleanup.enabled";
```

Comment:
Seems confusing is the consensus here. Maybe `java.cleanup.actionsOnSave` or `java.cleanup.commandsOnSave` ?

Reply:
I like `cleanup.actionsOnSave`, I think it's much better than just `enabled`. Thanks!